packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -51,7 +53,9 @@ test.describe('Call Composite E2E Configuration Screen Tests', () => {
   test('composite pages load completely', async ({ pages }) => {
```

Comment:
CallWithChatComposite also has a pre-join screen experience that should probably get this tooltip mitigation also